packages/create-app/templates/default-app/packages/app/package.json.hbs

```diff
@@ -56,6 +56,7 @@
     "@testing-library/jest-dom": "^5.10.1",
     "@testing-library/react": "^12.1.3",
     "@testing-library/user-event": "^14.0.0",
+    "@testing-library/dom": "^8.0.0",
```

Comment:
Should add `reacte-router` to deps here as well right? Like you did in `packages/app`

Reply:
aye! done